docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -90,6 +90,20 @@ cp gitea /usr/local/bin/gitea
 
 See how to create [Linux service]({{< relref "run-as-service-in-ubuntu.en-us.md" >}})
 
+## Updating to a new version
+
+You can update to a new version of gitea by stopping gitea, replacing the binary at `/usr/local/bin/gitea` and restarting the instance. 
```

Comment:
What if the gitea binary is at another location?

Reply:
@JonasFranzDEV installation from binary recommends this location so we should stick to this location

Reply:
Fair enough